# Surgical Outcomes: Both SPECT/CT and 4D CT Non-localizing

### Objective

Evaluate **surgical cure rates in patients with both negative SPECT/CT and 4D CT scans**, assessing outcomes when imaging fails to localize disease.

### Data Source

- `side_concordance_results.xlsx` containing pre-operative imaging results and surgical outcomes.

### Outputs

- Number and percentage of patients cured despite non-localizing imaging

- Interpretation notes for manuscript reporting
---


In [2]:
import pandas as pd 
preop_imaging_df = pd.read_excel('../../Data/side_concordance_results.xlsx')


In [3]:
# Define modalities
spect_modality = 'SPECT/CT'
modality_4dct = '4D CT Scan'

In [4]:
negative_spect_and_4d = preop_imaging_df[
    (preop_imaging_df['SPECT/CT'] == 'Non-localizing') &
    (preop_imaging_df['4D CT Scan'] == 'Non-localizing')
]
total_cases = negative_spect_and_4d.shape[0]
total_surgical_cure = negative_spect_and_4d[negative_spect_and_4d['Surgical Cure'] == 'Yes'].shape[0]
percentage = (total_surgical_cure / total_cases) * 100 

print(f"{total_cases} cases ({percentage:.2f}%) had both modalities non-localizing but were cured in surgery.")

negative_spect_4d_summary_df = pd.DataFrame({
    'Metric': [
        'Total cases with both modalities non-localizing',
        'Cases cured in surgery',
        'Percent cured within both-negative cases'
    ],
    'Count': [
        total_cases,
        total_surgical_cure,
        f"{percentage:.2f}%"
    ]
})
display(negative_spect_4d_summary_df)

16 cases (93.75%) had both modalities non-localizing but were cured in surgery.


Metric   Count
0  Total cases with both modalities non-localizing      16
1                           Cases cured in surgery      15
2         Percent cured within both-negative cases  93.75%

In [5]:
negative_spect_and_4d

Procedure_Date  Patient_ID Age  Gender              Surgical_Findings  \
156     2019-10-22      212628  NaN      F  Left Superior, Right Inferior   
162     2020-02-18      260846   56      F                 Right Inferior   
181     2020-10-20      167447   68      F                 Right Inferior   
223     2021-11-23      276774   58      M            No Tumor Identified   
226     2021-12-10      278475   60      M                  Left Inferior   
232     2022-01-20      280889   51      F                  Left Superior   
234     2022-02-10      279828   70      M                 Right Inferior   
252     2022-05-24      279513   69      F            4 Gland Hyperplasia   
262     2022-07-14      244595   70      F                  Left Superior   
264     2022-07-21      282031   71      F  Left Superior, Right Inferior   
277     2022-10-11      282887   47      F                  Left Superior   
302     2023-03-21      270719   50      M  Left Superior, Right Superior   
332     2023-09-12      292756   44      F                 Right Inferior   
337     2023-10-17      294431   35      M                 Right Inferior   
345     2023-12-19      285487   52      F                  Left Superior   
382     2025-01-21      307537   50      F                 Right Inferior   

    Surgical Cure        SPECT/CT      Ultrasound      4D CT Scan Sestamibi  \
156           Yes  Non-localizing             NaN  Non-localizing       NaN   
162           Yes  Non-localizing  Non-localizing  Non-localizing       NaN   
181           Yes  Non-localizing             NaN  Non-localizing       NaN   
223            No  Non-localizing   Left Inferior  Non-localizing       NaN   
226           Yes  Non-localizing             NaN  Non-localizing       NaN   
232           Yes  Non-localizing             NaN  Non-localizing       NaN   
234           Yes  Non-localizing             NaN  Non-localizing       NaN   
252           Yes  Non-localizing  Non-localizing  Non-localizing       NaN   
262           Yes  Non-localizing             NaN  Non-localizing       NaN   
264           Yes  Non-localizing             NaN  Non-localizing       NaN   
277           Yes  Non-localizing             NaN  Non-localizing       NaN   
302           Yes  Non-localizing             NaN  Non-localizing       NaN   
332           Yes  Non-localizing             NaN  Non-localizing       NaN   
337           Yes  Non-localizing             NaN  Non-localizing       NaN   
345           Yes  Non-localizing  Non-localizing  Non-localizing       NaN   
382           Yes  Non-localizing  Non-localizing  Non-localizing       NaN   

     MRI Ultrasound_Concordance SPECT/CT_Concordance 4D CT Scan_Concordance  \
156  NaN                No Scan                   No                     No   
162  NaN                     No                   No                     No   
181  NaN                No Scan                   No                     No   
223  NaN                     No           Yes (Side)             Yes (Side)   
226  NaN                No Scan                   No                     No   
232  NaN                No Scan                   No                     No   
234  NaN                No Scan                   No                     No   
252  NaN             Yes (Side)           Yes (Side)             Yes (Side)   
262  NaN                No Scan                   No                     No   
264  NaN                No Scan                   No                     No   
277  NaN                No Scan                   No                     No   
302  NaN                No Scan                   No                     No   
332  NaN                No Scan                   No                     No   
337  NaN                No Scan                   No                     No   
345  NaN                     No                   No                     No   
382  NaN                     No                   No                     No   

    Se

---

### Interpretation

- **Total cases:** 16 patients had **both SPECT/CT and 4D CT scans non-localizing**.

- **Surgical cure rate:** 15 out of 16 patients (**93.75%**) were cured at surgery despite negative imaging.

> **Clinical implication:** A high surgical cure rate (**~94%**) was achieved even when both imaging modalities were non-localizing, suggesting that **negative imaging does not preclude successful operative management**, likely due to strong intraoperative assessment and surgeon expertise.



### **Next Steps**

- Compare these findings with cure rates in patients who had positive imaging to guide **pre-operative imaging utility discussions**.